examples/port-2-aws-connection/versions.tf

```diff
@@ -5,5 +5,9 @@ terraform {
       source  = "equinix/equinix"
       version = ">= 1.20.0"
```

Comment:
Versions need to be updated to be 1.25.1 to include the bug fix for aws integration.